data/dummydata.v2.js

```diff
@@ -1,1724 +1,48 @@
-  { "id": 423, "name": 'Orale Orale', "about":{"description": "", "phone": "", "price": "enum", "style": "", "hours": "" }, "banner": "", "photo": ["https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta1.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta2.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta3.jpg"]},
-  { "id": 305, "name": 'Pacific Catch', "about":{"description": "", "phone": "", "price": "enum", "style": "", "hours": "" , "banner": "", "photo": ["https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta1.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta2.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta3.jpg"]}}
-];
-
-
-
-
-var randomElement = function(array){
-  var randomIndex = Math.floor(Math.random() * array.length)
+const randomElement = function(array) {
+  const randomIndex = Math.floor(Math.random() * array.length);
   return array[randomIndex];
-}
+};
 
-var price = ["$", "$$", "$$$", "$$$$", "$$$$$"];
-var styles = ['fancy', 'casual', 'dress-code enforced', 'no dress-code'];
-var hours = ["M - F, 7am - 8pm", "M - Sat, 10am - 9pm", "M - Sun, 8am - 10pm", "M - F, 10am - 6pm"];
+const price = ["$", "$$", "$$$", "$$$$", "$$$$$"];
+const styles = ['fancy', 'casual', 'dress-code enforced', 'no dress-code'];
+const hours = ["M - F, 7am - 8pm", "M - Sat, 10am - 9pm", "M - Sun, 8am - 10pm", "M - F, 10am - 6pm"];
 
 
-var ext = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/ext1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext3.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext4.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext5.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext6.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext7.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext8.jpeg"];
-var int = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/int1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int3.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int4.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int5.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int6.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int7.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/int8.jpeg"];
+const ext = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/ext1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext3.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext4.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext5.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext6.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext7.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/ext8.jpeg"];
```

Comment:
consider naming this variable 'exterior' and (below) 'interior' so that it's a little more clear what they are

Reply:
updated as well